cctp_primitives/src/bit_vector/compression.rs

```diff
@@ -14,6 +14,12 @@ use crate::{
     type_mapping::Error,
 };
 
+/// The chunk size used in the decompression functions.
+const DECOMPRESSION_CHUNK_SIZE: usize = 1024;
+
+/// The maximum size [bytes] allowed for decompressed buffers.
+/// When decompressing, if the size exceeds this threshold, an error is returned. 
+const MAX_DECOMPRESSION_SIZE: usize = 1024 * 260; // 260 KB
```

Comment:
It might be useful add a comment with the rationale of this value, which has been choosen as the double of the current limit in MC, which in turn is 2^12*254 bits

Reply:
Fixed.